src/Bridges/ApplicationLatte/TemplateFactory.php

```diff
@@ -83,6 +83,12 @@ public function createTemplate(UI\Control $control = NULL)
 		$latte->addFilter('modifyDate', function ($time, $delta, $unit = NULL) {
 			return $time == NULL ? NULL : Nette\Utils\DateTime::from($time)->modify($delta . $unit); // intentionally ==
 		});
+		$latte->addFilter('translate', function ($text/*, ...$params*/) use ($template) {
+			if (($translator = $template->getTranslator()) && $translator instanceof Nette\Localization\ITranslator) {
+				return call_user_func_array([$translator, 'translate'], func_get_args());
+			}
+			throw new Nette\InvalidStateException('Translator has not been set.');
+		});
```

Comment:
* master is PHP 5.6+, so you use `...$params`
* the `instanceof` check is useless, because `setTranslator` already handles this
* translate filter is **performance critical** (because it can be called 100×/request), this change makes it a lot slower, you would need to extract the check outside of translate call